pkg/devfile/generator/generators_test.go

```diff
@@ -168,11 +166,14 @@ func TestGetContainers(t *testing.T) {
 			wantContainerName:  containerNames[0],
 			wantContainerImage: containerImages[0],
 			wantContainerEnv: []corev1.EnvVar{
-
 				{
 					Name:  "PROJECTS_ROOT",
 					Value: "/myroot",
 				},
+				{
+					Name:  "PROJECT_SOURCE",
```

Comment:
```suggestion
					Name:  "PROJECT_SOURCE",
```
We don't need this `PROJECT_SOURCE`